### DISCUSSION TITLE
Another Pull Request

### DISCUSSION THREAD

**Discussion by XichengSpencer** [oppia/oppia-android#4968] [Q&A (Other)] [answered]
If my previous pull request for issue A is waiting for approval, I want to work on a new issue B.  Should I create a new branch from the develop branch and only commit changes for issue B, or should I create a new branch from my branch for A to contain the changes for both issue A and issue B?

**Comment by adhiamboperes** [comment, marked as answer]
Hi @XichengSpencer, please create a new branch from develop, so that your new PR only has changes related to the current issue.
